src/pip/_internal/operations/install/wheel.py

```diff
@@ -52,6 +52,10 @@
 
 def normpath(src, p):
     # type: (str, str) -> str
+    # for Windows: if paths start with different disk symbols,
+    # do not handle reletive paths
```

Comment:
Perhaps this can be reworded as `Do not handle relative paths if they belong to different logical disks on Windows`

Reply:
I don't know why but I also prefer the reworded version :smile:

Reply:
I'll reword it. :)